Makefile

```
@@ -0,0 +1,207 @@
+
+# Project specific variables
+CLIENT_LANDING_FILTER := --filter @loomify/marketing
+CLIENT_WEBAPP_FILTER := --filter @loomify/webapp
+CLIENT_DOCS_FILTER := --filter @loomify/docs
+
+# ------------------------------------------------------------------------------------
+# HELP
+# ------------------------------------------------------------------------------------
+
+# Provides help information for the available commands.
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (33).

(maxbodylength)